omaha/omaha_test.go

```diff
@@ -1,137 +0,0 @@
-package omaha
-
-import (
-	"encoding/xml"
-	"fmt"
-	"io/ioutil"
-	"os"
-	"testing"
-)
-
-func TestOmahaRequestUpdateCheck(t *testing.T) {
-	file, err := os.Open("../fixtures/update-engine/update/request.xml")
```

Comment:
I'm not sure why you are moving this to inline. I'd prefer a standalone fixture, but I guess you have some reasons to do this. Should the original `request.xml` stay around then?